frontend/components/events/create/forms/ReservedSlotsForm.tsx

```
@@ -0,0 +1,200 @@
+  const handleKeyPress = (e: React.KeyboardEvent) => {
+    if (e.key === "Enter") {
+      e.preventDefault();
+      handleAddSlot();
+    }
+  };
+
+  return (
+    <div className="mt-6">
+      <div className="flex items-center gap-2 mb-3">
+        <IconUsers size={20} className="text-gray-600" />
+        <label className="text-black text-lg font-semibold">Reserved Slots</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)